sklearn/datasets/samples_generator.py

```diff
@@ -94,10 +96,13 @@ def make_classification(n_samples=100, n_features=20, n_informative=2,
         exceeds 1.
 
     flip_y : float, optional (default=0.01)
-        The fraction of samples whose class are randomly exchanged.
+        The fraction of samples whose class are randomly exchanged. Larger 
```

Comment:
Please remove trailing white spaces, it is not PEP8.

You have three of them according to Travis

Reply:
Sorry about that, thanks.